src/helm/benchmark/run_spec_factory.py

```diff
@@ -141,9 +148,22 @@ def alter_run_spec(run_spec: RunSpec) -> RunSpec:
         if MISTRAL_MODEL_TAG in model.tags:
             run_spec = singleton(MistralRunExpander().expand(run_spec))
 
-        # IDEFICS instruct
-        if IDEFICS_INSTRUCT_MODEL_TAG in model.tags:
-            run_spec = singleton(IDEFICSInstructRunExpander().expand(run_spec))
+        # Google Gemini Vision returns an empty completion or throws an error if max_tokens is 1
```

Comment:
This change was already in `main` but got accidentally removed during a refactor.